tasks/libs/pipeline_tools.py

```diff
@@ -35,7 +45,17 @@ def cancel_pipelines_with_confirmation(gitlab, pipelines):
             ),
         )
 
-        print(color_message("Started at", "blue"), pipeline['created_at'])
+        try:
+            pipeline_creation_date = pipeline['created_at']
+            print(
+                "{} {:%c} ({})".format(
+                    color_message("Started at", "blue"),
+                    parse_datetime(pipeline_creation_date).astimezone(),
+                    pipeline_creation_date,
+                )
+            )
+        except ValueError:
```

Comment:
when does this happen?

Reply:
This can happen if the date is malformed. The idea behind this was to be sure I'm not breaking anything, if you think it's unnecessary I would be happy to remove it.

Reply:
I see, I would rather have it break if something goes wrong so that we notice; it will be easier to debug with an explicit traceback message